### ISSUE TITLE
[PeoplePicker] "Delete" button on resolved suggestion does not have an ARIA label

### ISSUE THREAD

**Issue by martellaj** [OfficeDev/office-ui-fabric-react#2279]
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: High (Accessibility) 

Products/sites affected: ODSP, any consumer of web UI

#### Describe the issue:

Resolved suggestions in the PeoplePicker component have an "X" button to remove the suggestion, but that "X" button doesn't have an ARIA label.

#### Actual behavior:

Screen reader does not describe purpose of button.

#### Expected behavior:

Screen reader describes purpose of button.
